.gitignore

```diff
@@ -588,4 +588,6 @@ MigrationBackup/
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
 
-# End of https://www.toptal.com/developers/gitignore/api/dotnetcore,visualstudio,visualstudiocode,python,rider
\ No newline at end of file
+# End of https://www.toptal.com/developers/gitignore/api/dotnetcore,visualstudio,visualstudiocode,python,rider
+appsettings.json
```

Comment:
Maybe not exclude app settings from source control? It will be easier to have this committed as a null value json to not expose secrets and keys :)